test/functional/http/get/transactions.js

```diff
@@ -768,8 +768,9 @@ describe('GET /api/transactions', function () {
 
 		it('using unknown id should be ok', function () {
```

Comment:
Why it should be ok?

Reply:
We no longer throw an error when the response is empty in the new API design.